java/client/src/main/java/glide/api/commands/PubSubClusterCommands.java

```diff
@@ -46,4 +47,70 @@ public interface PubSubClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> publish(GlideString message, GlideString channel, boolean sharded);
+
+    /**
+     * Lists the currently active shard channels.
+     *
+     * @apiNote When in cluster mode, the command is routed to all nodes, and aggregates the response
+     *     into a single array.
```

Comment:
Not sure that a user should know such details

Reply:
Futhermore, this command exists in cluster only

Reply:
will remove this then